docs/users/cli-usage.md

```diff
@@ -130,11 +130,11 @@ If necessary, run build steps that you may require, like `composer install` in t
 
 **Composer Setup Example**
 
-*Do not use this technique with Windows, as composer on an NTFS filesystem can result in major problems with NTFS symlinks that can't be read in the Linux web container. Instead, use the [community-provided hints on Stack Overflow](https://stackoverflow.com/questions/49660082/how-can-i-run-composer-with-ddev)*
-
 ```
-composer create-project typo3/cms-base-distribution my-typo3-site "^9"
+mkdir my-typo3-site
 cd my-typo3-site
+ddev config --project-type php
+ddev composer create-project typo3/cms-base-distribution . ^9
```

Comment:
👍 I only wonder if the "." will confuse people, but not sure of a better way. Could use /var/www/html there. I guess let's just maybe add an explanation below it? Maybe 
```
# This does "composer create-project" in the current directory (".") (inside the container)
```

Reply:
Doesn't the Drupal 8 composer example need this also?

Reply:
Probably need `ddev config --project-type typo3 --php-version 7.2` after that stuff.

Reply:
And it probably needs an extra `ddev start` after the config, so it gets the right nginx config.